packages/@react-spectrum/searchfield/docs/SearchField.mdx

```diff
@@ -171,6 +171,30 @@ By default, the label is positioned above the SearchField. The `labelPosition` p
 <SearchField label="Search" labelPosition="side" labelAlign="end" />
 ```
 
+### Help text
+[View guidelines](https://spectrum.adobe.com/page/help-text/#Usage-guidelines)
+
+Both a description and an error message can be supplied to a SearchField. Description shows all the time except when the `validationState` is "invalid". The error message should help the user fix their input quickly and it should be changed depending on the detected error. Both the description and error message should be localized as well.
+
+```tsx example
+function Example() {
+  let [value, setValue] = React.useState('0');
+  let isValid = React.useMemo(() => /^\d$/.test(value), [value]);
+
+  return (
+    <SearchField
+      validationState={isValid ? 'valid' : 'invalid'}
+      value={value}
+      onChange={setValue}
+      label="Search your favorite number"
+      maxLength={1}
+      description="Enter a single digit number."
+      errorMessage={value === '' ? 'Empty input not allowed.' : 'Single digit numbers are 0-9.'}
```

Comment:
The only way I got the second error message was to enter a negative sign. Are there other ways? I'm thinking about usability for people reading the docs.

Reply:
any letter will do it